src/Ng/Ng.csproj

```diff
@@ -61,6 +61,10 @@
       <HintPath>..\..\packages\HtmlAgilityPack.1.4.9\lib\Net45\HtmlAgilityPack.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Hyak.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
KeyVault..

---
In reply to: [70754613](https://github.com/NuGet/NuGet.Services.Metadata/pull/77#discussion_r70754613) [](ancestors = 70754613)